common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -22,12 +22,11 @@
     % for option in options:
         <li>
             % if has_options_value:
-                % if all([c == 'correct' for c in option['choice'], status]):
-                <span class="tag-status correct" id="status_${id}" aria-describedby="input_${id}_comment"><span class="sr">Status: Correct</span></span>
-                % elif all([c == 'partially-correct' for c in option['choice'], status]):
-                <span class="tag-status partially-correct" id="status_${id}" aria-describedby="input_${id}_comment"><span class="sr">Status: Partially Correct</span></span>
-                % elif all([c == 'incorrect' for c in option['choice'], status]):
-                <span class="tag-status incorrect" id="status_${id}" aria-describedby="input_${id}_comment"><span class="sr">Status: Incorrect</span></span>
+                % if all([c == status.classname for c in option['choice'], status]):
+                <span class="tag-status correct" aria-describedby="input_${id}_comment">
```

Comment:
This shouldn't still have "correct" as a class (needs to have status.classname). Have you been able to test this problem type at all? It would be great to have some confidence that this refactoring worked. You could look at tests of annotation to get some ideas about how to hack one together.